src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLumberjack.java

```diff
@@ -225,6 +246,20 @@ public void deserializeNBT(final CompoundNBT compound)
         {
             endRestriction = null;
         }
+
+        int i = 0;
+        while (true)
```

Comment:
Check how we write blockPos to nbt in some other buildings, there are tag lists that are much easier to use than this construct

Reply:
BlockPosUtil